projects/1.20/assets/blue-skies/blue_skies/lang/zh_cn.json

```
@@ -0,0 +1,1354 @@
+	"subtitles.entity.arachnarch.lunge": "幽咽蛛王：猛扑",
+	"subtitles.entity.arachnarch.flip": "幽咽蛛王：向下翻转",
+	"subtitles.entity.arachnarch.sling": "幽咽蛛王：向上倒悬",
+	"subtitles.entity.arachnarch.stunned": "幽咽蛛王：晕眩地咕哝",
+	"subtitles.entity.arachnarch.venom_drop": "幽咽蛛王：排泄",
+	"subtitles.entity.arachnarch.screech": "幽咽蛛王：尖叫",
+	"subtitles.entity.arachnarch.attack": "幽咽蛛王：攻击",
+	"subtitles.entity.arachnarch.throw": "幽咽蛛王：投掷",
+	"subtitles.entity.nested_spider.idle": "巢穴蜘蛛：嘶嘶",
+	"subtitles.entity.nested_spider.hurt": "巢穴蜘蛛：受伤",
+	"subtitles.entity.nested_spider.death": "巢穴蜘蛛：死亡",
+	"subtitles.entity.boss.spawn": "Boss：生成",
+	"subtitles.entity.fluctuant_sphere.disappear": "波动球体：消散",
+	"subtitles.entity.sky_player.spit": "玩家：发射毒液",
+	"subtitles.entity.sky_player.cast_spell": "玩家：施法",
+	"subtitles.entity.sky_player.prepare_rocks": "玩家：准备岩块",
+	"subtitles.entity.sky_player.prepare_fluctuant_sphere": "玩家：准备波动球体",
+	"subtitles.entity.sky_player.prepare_summoning": "玩家：准备召唤",
+	"subtitles.entity.sky_player.prepare_hammer": "玩家：举锤",
+	"subtitles.entity.sky_player.hammer_smash": "玩家：重击",
+	"subtitles.entity.villager.work_stargazer": "观星者：工作",
+	"subtitles.entity.villager.work_wrangler": "牧人：工作",
+	"subtitles.entity.villager.work_shoveler": "铲雪人：工作",
+	"subtitles.entity.villager.work_nightwatcher": "守夜人：工作",
+	"subtitles.entity.villager.work_summoner": "召唤师：工作",
+	"subtitles.entity.villager.work_alchemist": "炼金师：工作",
+
+	"advancements.blue_skies.root": "蔚蓝浩空",
+	"advancements.blue_skies.root.desc": "蔚蓝浩空模组。",
+	"advancements.blue_skies.everbright.enter": "永昼？",
+	"advancements.blue_skies.everbright.enter.desc": "找到永昼之地传送门并进入其中。",
+	"advancements.blue_skies.everdawn.enter": "恒晓？",
+	"advancements.blue_skies.everdawn.enter.desc": "找到恒晓之地传送门并进入其中。",
+	"advancements.blue_skies.thinking_with_portals": "现在，你懂得如何使用传送门了！",
+	"advancements.blue_skies.thinking_with_portals.desc": "进入永昼之地和恒晓之地这两个维度。",
+	"advancements.blue_skies.everbright.infused": "能量小子！",
+	"advancements.blue_skies.everbright.infused.desc": "在地牢中找到一把注能剑。",
+	"advancements.blue_skies.everdawn.think_different": "确实奇异！",
+	"advancements.blue_skies.everdawn.think_different.desc": "在猛毒地牢中找到一把奇异之剑。",
+	"advancements.blue_skies.everdawn.poison_shroom": "对人杀手",
+	"advancements.blue_skies.everdawn.poison_shroom.desc": "踩到一株邪恶菇，感受它带给你的痛苦。",
+	"advancements.blue_skies.everdawn.kill_bug": "杀虫大师！",
+	"advancements.blue_skies.everdawn.kill_bug.desc": "杀死恒晓之地中所有种类的敌对昆虫和蜘蛛。",
+	"advancements.blue_skies.everdawn.get_wood": "新的黎明",
+	"advancements.blue_skies.everdawn.get_wood.desc": "获得一块恒晓之地的原木。",
+	"advancements.blue_skies.everbright.get_wood": "光明未来！",
+	"advancements.blue_skies.everbright.get_wood.desc": "获得一块永昼之地的原木。",
+	"advancements.blue_skies.blue_sky_pie": "我的~，我的~，蔚蓝天空下的派~！",
+	"advancements.blue_skies.blue_sky_pie.desc": "制作一份彗星莓派。",
+	"advancements.blue_skies.regrowth": "可再生资源",
+	"advancements.blue_skies.regrowth.desc": "获得彗星树苗。",
+	"advancements.blue_skies.everbright.regen_brewberry": "粉色娇嫩",

```

```suggestion
	"advancements.blue_skies.everbright.regen_brewberry": "红粉佳人",
```
[红粉佳人（豆瓣）](https://movie.douban.com/subject/1297376/)